content/documentation/pages/5-feature-guides/1-streams/2-fanin-fanout.md

```diff
@@ -4,6 +4,27 @@ title: 'Fan-in and Fan-out'
 description: 'How to support fan-in and fan-out use cases.'
 ---
 
-Introduction to fan-in fan out.
+# Fan-in and Fan-out
 
-Look at ref guide - migrate content?
+By using named destinations, you can support fan-in and fan-out use cases.
```

Comment:
"named destinations" can be linked to the ref. guide docs. I think: http://docs.spring.io/spring-cloud-dataflow/docs/2.0.2.RELEASE/reference/htmlsingle/#spring-cloud-dataflow-stream-dsl-named-destinations

If we are moving this section also to the Microsite, we can link to the relative path instead.